ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -305,10 +305,7 @@ private static List<ColumnName> implicitColumns(
         .collect(Collectors.toList());
 
     if (fieldNames.size() != values.size()) {
-      throw new KsqlException(
-          "Expected a value for each column."
-              + " Expected Columns: " + fieldNames
-              + ". Got " + values);
+      throw new KsqlException("Expected a value for each column.");
```

Comment:
nit: it'd be good to include `fieldNames.size()` and `values.size()` in this error message, to make it more informative.